netutils/config/parser.py

```diff
@@ -331,6 +331,100 @@ def build_config_relationship(self) -> t.List[ConfigLine]:
             self._update_config_lines(line)
         return self.config_lines
 
+    @staticmethod
+    def _match_type_check(line: str, pattern: str, match_type: str) -> bool:
+        """Checks pattern for exact match or regex."""
+        if match_type == "exact" and line == pattern:
+            return True
+        if match_type == "startswith" and line.startswith(pattern):
+            return True
+        if match_type == "endswith" and line.endswith(pattern):
+            return True
+        if match_type == "regex" and re.match(pattern, line):
+            return True
+        return False
+
+    def find_all_children(self, pattern: str, match_type: str = "exact") -> t.List[str]:
+        """Returns configuration part for a specific pattern not including parents.
+
+        Args:
+            pattern: pattern that describes parent.
+            match_type (optional): Exact or regex. Defaults to "exact".
+
+        Returns:
+            configuration under that parent pattern.
+        Example:
+            >>> config = '''
+            ... router bgp 45000
+            ...   address-family ipv4 unicast
+            ...    neighbor 192.168.1.2 activate
+            ...    network 172.17.1.0 mas'''
```

Comment:
Not following this command? Missing `mask`?

Reply:
Changed.